oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -742,6 +750,10 @@ int mca_spml_ucx_put_nb(shmem_ctx_t ctx, void* dst_addr, size_t size, void* src_
     status = ucp_put_nbi(ucx_ctx->ucp_peers[dst].ucp_conn, src_addr, size,
                      (uint64_t)rva, ucx_mkey->rkey);
 
+    if ((status >= 0) && mca_spml_ucx.synchronized_quiet) {
+        ucx_ctx->ucp_peers[dst].put_op = 1;
```

Comment:
can we keep this is a separate bitmap to avoid cache thrashing?

Reply:
done